transcendence-app/seeds/003_seed_games.ts

```diff
@@ -24,9 +32,14 @@ const seedGames = async (knex: Knex) => {
     .from('users')
     .whereIn('username', defaultUsernames);
   const getRandomUser = () => users[Math.floor(Math.random() * users.length)];
-  const games = Array.from({ length: GAMES_NUMBER }, () =>
-    createRandomGame(getRandomUser().username, getRandomUser().username),
-  );
+  const games = Array.from({ length: GAMES_NUMBER }, () => {
+    const usernameOne = getRandomUser().username;
+    let usernameTwo = getRandomUser().username;
+    while (usernameTwo === usernameOne) {
```

Comment:
Like this we always get GAMES_NUMBER number of records, and we don't have to go around filtering, and making exceptions.